docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-114.adoc

```diff
@@ -0,0 +1,47 @@
+== Clarifai API Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_114
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+A Clarifai API Key is vital for accessing Clarifai's powerful AI and machine learning services, which provide advanced image and video recognition capabilities. This key is crucial for authenticating requests to the API and ensuring that only authorized clients can utilize the services. Exposure of this key can lead to unauthorized access, misuse of the API, and potential data leaks. Securely managing this key is essential to maintain the security and integrity of your applications. Regularly reviewing your security settings and practices is important to prevent unauthorized usage. For detailed security guidelines and best practices, refer to the latest https://docs.clarifai.com/clarifai-basics/authentication/personal-access-tokens/[Clarifai documentation and support] resources.
+
+=== Fix - Buildtime
+
+*Clarifai*
+
+If your Clarifai API Key is exposed, immediate action is required to revoke the compromised key and generate a new one to secure access to the API. Follow these steps to replace the exposed key and update all dependent applications:
+
+1. Log into your Clarifai account using your administrator credentials.
+
+2. Navigate to the 'API Keys' section within your account settings.
+
+3. Identify the exposed key and delete it.
+
+4. Generate a new API key by selecting 'Create new key'. Provide a name for the new key and save the changes.
```

Comment:
Select 'Create new key' to generate a new API Key. Provide a name for the new key and save the changes.